### PR TITLE
Merge release r1.20.0 to main

### PR DESCRIPTION
**Pull request by ericharper**
# What does this PR do ?

Make sure all bugfixes and doc updates were properly cherry picked to main.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: docs/source/starthere/tutorials.rst [hunk 1]
```diff
@@ -107,14 +107,8 @@ To run a tutorial:
...
    * - ASR
-     - Hybrid ASR-TTS Models
-     - `Hybrid ASR-TTS Models <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/ASR_TTS_Tutorial.ipynb>`_
-   * - ASR
-     - ASR Confidence Estimation
-     - `ASR Confidence Estimation <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/ASR_Confidence_Estimation.ipynb>`_
-   * - ASR
-     - Confidence-based Ensembles
-     - `Confidence-based Ensembles <https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/asr/Confidence_Ensembles.ipynb>`_
+     - Hybrid ASR-TTS Models Tutorial
```
Comment (titu1994) [inline docs/source/starthere/tutorials.rst:110]: Hmm  why is ASR Confidence Estimation being removed ? Plus Confidence based Ensembles. @Kipok is this intentional?

Comment (Kipok) [reply docs/source/starthere/tutorials.rst:110]: Not sure about the Hybrid models, but confidence tutorials should not be removed. I think there was a PR that mistakenly removed them and maybe it was only fixed in main and not in the release branch?

Comment (ericharper) [reply docs/source/starthere/tutorials.rst:110]: It might need to be reverted I could have resolved the conflict incorrectly

Comment (titu1994) [reply docs/source/starthere/tutorials.rst:110]: @ericharper  seems this removal was a mistake. Can you add back these deleted rows and keep the added items?

**Review by titu1994 — APPROVED** [review]
Thanks !

**Pull request merged**